### DISCUSSION TITLE
[Question]: Is it possible to activate plugins for both OpenAI and Azure OpenAI?

### DISCUSSION THREAD

**Discussion by dividor** [danny-avila/LibreChat#2695] [Q&A] [answered]
### What is your question?

Is it possible to activate plugins for both OpenAI and Azure OpenAI, if so how do we do that in the librechat.yaml?

### More Details

I have a `librechat.yaml` configured with Azure OpenAI custom endpoint, with plugins set to true. Works really nicely.

However, I couldn't quite see how to also activate plugins for OpenAI in `librechat.yaml`. I know there is a legacy `PLUGIN_MODELS` in the `.env` config, but I thought from the documentation that this is perhaps deprecated.

 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
No, but allowing for both will be part of a future update

**Reply by dividor** [reply]
Thanks for confirming.
